readme/api/references/plugin_manifest.md

```diff
@@ -39,6 +40,15 @@ Name | Type | Required? | Description
 | src | a relative path to src dir. |
 | label | description of the image. |
 
+## Icons
+| Properties | Description |
+| --- | --- |
+| 16 | a relative path to an image .png |
+| 32 | a relative path to an image .png |
+| 48 | a relative path to an image .png |
+| 128 | a relative path to an image .png |
+You can declare as many key/value pairs icons you wish, as long as, you declare these four sizes defined above. 
```

Comment:
Could you check how it renders please? For example I see that this last line is part of the table. You need to add a new line above it to prevent this.

Also I don't understand what this sentence mean? What other key/value pairs is the user going to define?